### ISSUE TITLE
Filter RSS Feeds by category

### ISSUE THREAD

**Issue by dreamflasher** [eikowagenknecht/lootscraper#214]
### Is your feature request related to a problem? Please describe.

Like the telegram bot, I'd like to filter the RSS feed by categories as well. 

### Describe the solution you'd like

Either provide a feed per category or add the category as a meta tag in the full RSS so that we can filter on it with external tools.

**Comment by eikowagenknecht** [comment]
There are already feeds per category:

https://github.com/eikowagenknecht/lootscraper#rss-feeds

What exactly would you want to filter by?

**Comment by dreamflasher** [comment]
My understanding of categories is Strategy, RPG, turn-based etc. – the tags/categories you have for the different providers (that's what I would understand as Steam, Amazon, Epic etc.)

**Comment by eikowagenknecht** [comment]
Ah okay, get it. And which Telegram board do you refer to?

**Comment by dreamflasher** [comment]
Typo, I meant bot. My understanding was that the bot can do this, which I didn't check. So I guess you can close the issue, assuming that you have no way to get the categories for the different providers?

**Comment by eikowagenknecht** [comment]
The game info (eg genres) is scraped from Steam and IGDB. So for most games the information should be there. There are too many different feeds there already, though. Adding it as meta data in the feeds when it’s available should be easy enough. Do you have any example of how it needs to look so that it works for your rss reader?

**Comment by dreamflasher** [comment]
Happy to hear that!
Yep, genres is a better name for what I've been looking for.
Also epic calls it genres, but I believe they are not in the RSS feed for Epic, yet? Eg. https://store.epicgames.com/en-US/p/terraforming-mars-18c3ad

I'd put it in the category tag of the items, see specification here:
https://cyber.harvard.edu/rss/rss.html#ltcategorygtSubelementOfLtitemgt
(one category per genre, you could also add the store (your category) as an additional category)

I'd filter it with huginn, but as the category is a standard in RSS there should exist many tools being able to filter based on category.

**Comment by eikowagenknecht** [comment]
I added the scraped genres from steam where available in the RSS feeds. Will be available on the public instance in about an hour.

**Comment by eikowagenknecht** [comment]
Example:

```
...
    <link href="https://gaming.amazon.com/once-upon-a-jester-fgwp-jun23/dp/amzn1.pg.item.704a2c1a-875f-48d2-bd8f-93168d19abdd?ingress=amzn&amp;ref_=SM_OUAJ01F_CRWN" rel="alternate"/>
    <category term="Genre: Adventure" scheme="https://store.steampowered.com/category/" label="Adventure"/>
    <category term="Genre: Casual" scheme="https://store.steampowered.com/category/" label="Casual"/>
    <category term="Genre: Indie" scheme="https://store.steampowered.com/category/" label="Indie"/>
    <published>2023-06-29T17:11:09.423974+00:00</published>
...
```

**Comment by dreamflasher** [comment]
Fantastic, thank you so much!
